gamestonk_terminal/stocks/discovery/ark_view.py

```diff
@@ -65,15 +65,15 @@ def ark_orders_view(
         df_orders = df_orders[df_orders.direction == "Buy"]
```

Comment:
The correct way to solve the warnings is to add .copy() to every slice here:
```suggestion
        df_orders = df_orders[df_orders.direction == "Buy"].copy()
```

Reply:
Thanks for the input! I was actually able to triangulate the issue to the ark_model. Somewhere in the code surrounded by the pandas warning mute functions. I tried adding this code to those but the warning still appeared. Do you have any further suggestions?

Reply:
I don't know the error here, but it's probably an error of the same type or due to using df[value][value2] insted of iloc or loc. The error is probably here I'm however not sure (line 110-113 or maybe in 102-103 due to not using iloc and loc):
![bild](https://user-images.githubusercontent.com/75195383/150698126-96d06ca1-33ff-408d-aa4c-ae54c186fd0e.png)